scripts/create_new_beamline.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+
+from nsls2api.infrastructure import mongodb_setup
+from nsls2api.infrastructure.config import get_settings
+from nsls2api.models.beamlines import Beamline, ServiceAccounts
+from nsls2api.services import pass_service
+
+settings = get_settings()
+
+# CHANGE THIS TO THE BEAMLINE YOU WANT TO CREATE
+BEAMLINE_NAME = "CDI"
+
+
+async def main():
+    # Initialize Beanie
+    await mongodb_setup.init_connection(settings.mongodb_dsn)
+
+    pass_resources = await pass_service.get_pass_resources()
+    pass_ids = [r["ID"] for r in pass_resources if r["Code"] == BEAMLINE_NAME]
+    if len(pass_ids) > 1:
+        raise ValueError(f"Multiple pass IDs found for {BEAMLINE_NAME}: {pass_ids}")
+
+    # CHANGE THESE VALUES TO SUIT YOUR NEEDS
+    new_beamline = Beamline(
+        name=BEAMLINE_NAME,
+        long_name="Fancy New Beamline",
```

Comment:
`long_name` is from the user office, correct?  I guess you do not use PASS because the beamline scientist can change this in PASS and break a bunch of stuff, right?